src/lib.rs

```diff
@@ -657,17 +673,14 @@ impl<T: Counter> Histogram<T> {
             min_non_zero_value: u64::max_value(),
 
             total_count: 0,
-            counts: Vec::new(), // set by alloc() below
+            // set by alloc() below
+            counts: Vec::new(),
         };
 
         // determine exponent range needed to support the trackable value with no overflow:
         let len = h.cover(high);
 
-        // Establish leading_zero_count_base, used in bucket_index_of() fast path:
-        // subtract the bits that would be used by the largest value in bucket 0.
-        h.leading_zero_count_base = (64 - h.unit_magnitude - h.sub_bucket_half_count_magnitude - 1) as u32;
```

Comment:
I may be wrong here, but this used to be below for a reason. If memory serves me right, `cover` re-calculated either `unit_magnitude` or `sub_bucket_half_count_magnitude`, and so this had to be computed after that. But that may have changed now?